### PR TITLE
single video Page Created

### PR DESCRIPTION
**Pull request by ivinayakg**
preview :- https://vide-x.netlify.app/video/ac1c6062-7261-44e6-8ebf-8879914576ad

### CODE REVIEW

File: src/components/VideoPlayer/index.js [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+      videoRef.current.currentTime += 10;
+      dispatch({ type: "PLAY", payload: true });
+    }
+  };
+  const backwardVideoBy10 = () => {
+    if (videoRef.current.currentTime !== 0) {
+      videoRef.current.currentTime -= 10;
+      dispatch({ type: "PLAY", payload: true });
+    }
+  };
```
Comment (rahulyadav139) [inline src/components/VideoPlayer/index.js:44]: This is great. You have implemented Forward and Backward functionality. Great work.

File: src/components/VideoPlayer/index.js [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+              <button onClick={playbackRateHandler} speed={1}>
+                Normal
+              </button>
+              <button onClick={playbackRateHandler} speed={1.25}>
+                1.25
+              </button>
+              <button onClick={playbackRateHandler} speed={1.5}>
+                1.5
+              </button>
+              <button onClick={playbackRateHandler} speed={2}>
```
Comment (rahulyadav139) [inline src/components/VideoPlayer/index.js:163]: Appreciation: You have implemented playback speed too. This is clean great work.

**Review by JowelTisso — APPROVED** [review]
* Clean code
* Small PR 
* Looks Good

**Review by rahulyadav139 — APPROVED** [review]
Everything is great. You have implemented so many features. Great work.

**Pull request merged**